bin/main.ml

```diff
@@ -150,7 +183,8 @@ let draw ui_terminal matrix (game_state : game_state) =
     (* draw letters box *)
     (let ctx = with_grid_cell ctx layout_spec 1 2 0 1 in
      let _ = with_frame ctx " letters " LTerm_draw.Heavy in
-     ());
+     ();
```

Comment:
these parens were just there to make the `let` statement work without containing anything. now that you are calling `draw_letters` inside of the let statement, the parens should be deleted